Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -69,10 +71,14 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
         super.viewDidLoad()
         tableView.addGestureRecognizer(longPressRecognizer)
 
-        self.tableView.accessibilityIdentifier = "Bookmarks List"
-        
-        self.refreshControl = UIRefreshControl()
-        self.tableView.addSubview(refreshControl!)
+        tableView.accessibilityIdentifier = "Bookmarks List"
+
+        if #available(iOS 11.0, *) {
+            tableView.dragDelegate = self
```

Comment:
All of these panels inherit from `SiteTableViewController` if you are up for it. I think you can simply this by build the implementation directly onto `SiteTableViewController`.